tests/php/modules/wpcom-block-editor/test_class-jetpack-wpcom-block-editor.php

```diff
@@ -1,5 +1,7 @@
 <?php
```

Comment:
Are we also planning to fix the tests before landing this one?

Reply:
yes!